code/modules/power/cell.dm

```diff
@@ -147,11 +147,11 @@
 		rigged = 1 //broken batterys are dangerous
 
 /obj/item/weapon/cell/emp_act(severity)
-	//remove this once emp changes on dev are merged in
-	if(isrobot(loc))
-		var/mob/living/silicon/robot/R = loc
-		severity *= R.cell_emp_mult

```

Comment:
Either tweak the cell_emp_mult to fulfil that new requirements or remove it.

Imho it should be adapted since the emp mult is currently changed by the overclock.